web/conf/routes

```diff
@@ -43,6 +43,12 @@ GET     /api/v1/party/groups                controllers.api.v1.User.getAllGroups
 
 GET     /api/v1/autocomplete/search         controllers.api.v1.Search.getSearchAutoComplete()
 
+GET     /api/v2/autocomplete/searchDataset         controllers.api.v1.Search.getSearchAutoCompleteForDataset()
+
+GET     /api/v2/autocomplete/searchMetric         controllers.api.v1.Search.getSearchAutoCompleteForMetric()
+
```

Comment:
Cool, I was referring to `getSearchAutoComplete()` seems like you had one route for two different controllers.
But I guess this is fine. thanks